blockchain/serviceMetadata.go

```diff
@@ -36,8 +36,87 @@ import (
             "endpoints": [
                 "https://tz-services-1.snet.sh:8005"
             ],
-            "group_id": "EoFmN3nvaXpf6ew8jJbIPVghE5NXfYupFF7PkRmVyGQ="
+            "group_id": "EoFmN3nvaXpf6ew8jJbIPVghE5NXfYupFF7PkRmVyGQ=",
+{
+  "licenses": {
+    "tiers": [{
+      "type": "Tier",
+      "planName": "Tier AAA",
+      "grpcServiceName": "ServiceA",
+      "grpcMethodName": "MethodA",
+      "range": [
+        {
+          "high": 100,
+          "fixedPriceInCogs": 1
+        },
+        {
+          "high": 200,
+          "fixedPriceInCogs": 0.75
+        },
+        {
+          "high": 300,
+          "fixedPriceInCogs": 0.50
+        }
+      ],
+      "detailsUrl": "http://abc.org/licenses/Tier.html",
+      "isActive": "true/false"
+    },
+     {
+      "type": "Tier",
+      "planName": "Tier BBB Applicable for All service.methods",
+      "range": [
+        {
+          "high": 100,
+          "fixedPriceInCogs": 1
+        },
+        {
+          "high": 200,
+          "fixedPriceInCogs": 0.75
+        },
+        {
+          "high": 300,
+          "fixedPriceInCogs": 0.50
         }
+      ],
+      "detailsUrl": "http://abc.org/licenses/Tier.html",
+      "isActive": "true/false"
+    }],
+    "subscriptions": {
+      "type": "Subscription",
+      "subscription": [
+        {
+          "periodInDays": 30,
+          "creditsInAGI": 120,
+          "planName": "Monthly For ServiceA/MethodA",
+          "LicenseCost": 90,
+          "grpcServiceName": "ServiceA",
+          "grpcMethodName": "MethodA"
+        },
+        {
+          "periodInDays": 30,
+          "creditsInAGI": 123,
+          "planName": "Monthly",
+          "LicenseCost": 93
```

Comment:
Why the LicenseCost field name different naming convention?

Reply:
@corrected this Sir